pkgs/applications/networking/browsers/chromium/darwin/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  makeDarwinApp,
+  fetchzip,
+  stdenv,
+}:
+let
+  specs = {
+    aarch64-darwin = {
+      arch = "Mac_Arm";
+      sha256 = "sha256-8FeYbXzMDoTVxeAaKe3F4SYRKDz2pirRj3BAF7gtCR8=";
+      version = "1169958";
+    };
+    x86_64-darwin = {
+      arch = "Mac";
+      sha256 = "sha256-Dje78MSnczxHIzJG7xXkTUqF/nsdSOAvpKhGU/rxoTg=";
+      version = "1171203";
+    };
+  };
+  spec = specs.${stdenv.hostPlatform.system} or (throw "Unknown platform");
+in
+makeDarwinApp rec {
+  appName = "Chromium";
+  binaryName = appName;
+  pname = "chromium";
+  version = spec.version;
+  src = fetchzip {
+    url = "https://commondatastorage.googleapis.com/chromium-browser-snapshots/${spec.arch}/${spec.version}/chrome-mac.zip";
+    sha256 = spec.sha256;
+  };
+
+  wrapBinary = true;
+  packageMeta = {
```

Comment:
Maybe we can set `hydraPlatforms = []`? Since we're just fetching an archive and unpacking it, having Hydra build and cache it isn't very useful.

Reply:
firefox-bin also does this FWIW.

Reply:
Done in [0ef984b](https://github.com/NixOS/nixpkgs/commit/0ef984bc970687f8df65205a836924c099d61906). Put it in `mkDarwinAppFromBin`